blog/2020/05/invalidations.md

```diff
@@ -0,0 +1,762 @@
+@def authors = "Tim Holy"
+@def published = "11 May 2020"
+@def title = "Analyzing sources of compiler latency in Julia: method invalidations"
+@def rss_pubdate = Date(2020, 5, 11)
+@def rss = """Julia is fast, but compiling Julia code takes time. This post analyzes why it's sometimes necessary to repeat that work, and what might be done to fix it."""
+
+
```

Comment:
You could (but don't have to) add a `\toc` here to insert a table of content. Use `@def mintoclevel = 2` and `@def maxtoclevel = 3` to control depth.

Btw all your section headers are level 1 (`#`) which will look a bit funny because the title of the blog post is level 1, you might want to increase the level of everything by one `#`